vllm/config/compilation.py

```diff
@@ -207,6 +211,9 @@ class CompilationConfig:
     splitting_ops: Optional[list[str]] = None
     """A list of ops to split the full graph into subgraphs, used in piecewise
     compilation."""
+    partition_rule_ops: list[str] = field(default_factory=list)
```

Comment:
My opinion is that we should just use `splitting_ops` directly here for simplicity. We could rename them if we want, but having two intertwined properties is more complex to think about.